pghoard/monitoring/prometheus.py

```diff
@@ -31,18 +29,16 @@ def unexpected_exception(self, ex, where, tags=None):
     def get_metrics(self):
         data = []
         for metric, value in self.metrics.items():
-            line = '{} {} {}'.format(metric, value.get("value"), value.get("ts"))
+            line = '{} {}'.format(metric, value.get("value"))
```

Comment:
Looks like `value` is no longer a `dict` but the raw value itself so the `.get("value")` part is invalid.